http3/server.go

```diff
@@ -321,12 +321,7 @@ func extractPort(addr string) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-
-	portInt, err := net.LookupPort("tcp", portStr)
-	if err != nil {
-		return 0, err
-	}
-	return portInt, nil
+	return net.LookupPort("tcp", portStr)
```

Comment:
no need to check if err the return  0 and err net.LookupPort is doing it in case of error net.LookupPort returns 0,err

Reply:
This is fine.